docs/reference/components/sinks/loki.cue

```diff
@@ -161,6 +161,15 @@ components: sinks: loki: {
 				"""
 		}
 
+		concurrency: {
+			title: "Concurrency"
+			body: """
+				To make sure logs arrive at Loki in a correct order,
+				the `loki` sink only sends one request at a time.
+				Setting `request.concurrency` will not have any effects.
```

Comment:
Should we just remove the option then? I realize that'll require a bit of rework internally since we share that struct for configuration, but I think it'd result in a better end-user experience.

It'd be nice to at least remove it from the docs.

Reply:
:+1: true, if there is a chance we'll add it back, we can keep it for now.